external/fv3fit/tests/training/test_train.py

```diff
@@ -203,6 +206,37 @@ def test_default_convolutional_model_is_transpose_invariant(regtest):
     xr.testing.assert_allclose(output_from_transpose, transpose_output, atol=1e-5)
 
 
+def test_diffusive_convolutional_model_gives_bounded_output():
+    """
```

Comment:
The means will not be the same because the mean of the input (when fit to the normalizer) is not the mean of the output. But given the input, the mean of the output *should* be almost exactly equal to the mean of the target output. Unfortunately this is not very worthwhile to test since this will be true of almost any mildly-trained model.